Sign_In_Page.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import Secure 
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.signInPage_url)
+    
+
+    def click_sign_in_button(self):
+        for cookie in Secure.cookies_coinbase: self.driver.add_cookie(cookie)
+        self.driver.find_element_by_id('signin_button').click()
```

Comment:
This needs a locator, put it outside the method. This way if another method needs this element and the locator ever changes we only need to edit it in one place. Once you have a locator, use `driver.find_element()`